data/wanderer/wanderer news.txt

```diff
@@ -0,0 +1,127 @@
+			"Gardener"
+	message
+		word
+			"You see a Wanderer carefully place some seeds in the ground. They then spread some sort of light grey powder over them."
+			"From the spaceport you can see some buildings with plants growing on their sides. A Wanderer prunes and waters them in mid-air."
+			"While walking through the spaceport, you see a large greenhouse full of various plants. A Wanderer inside is tending to them."
+			
+	name
+		word
+			"Group of Wanderers"
+	message
+		word
+			"While walking through the spaceport, you see several Wanderers flying through the air. You duck to avoid them."
+			"A crowd of Wanderers has formed around an object. Upon closer inspection, it appears to be some sort of technological device."
+			"As you walk by a group of Wanderers, they begin to coo and chitter at you. Evidently they have not seen something like you before."
+			
+news "Wanderers and Korath"
+	to show
+		has "Wanderers: Sestor: Final: done"
+	location
+			government "Wanderer"
+			not attributes "evacuation"
+	name 
+		word
+			"Wanderer merchant captain"
+			"Wanderer captain"
+	message
+		word
+			"The Korath have [enormous, gigantic] ships that many of them live on their whole lives. I cannot [imagine, fathom] doing the same."
+			"Have you [seen, viewed] a Korath up close? They only have feathers on their neck. How [strange, odd]."
+			"The engines the Korath use have no regard for [effciency, subtlety]. But I must admit, they are much more powerful than ours."
+			"The Korath cannot pay [well, adequately] for their transport jobs, but they insist on paying [despite it, nonetheless].
+			"I am trying to [learn, understand] more words of the Korath language. It is a difficult [task, undertaking], but it is important to learn the language of [those we work with, our peers]."
+	name
+		word
+			"Wanderer linguist"
+			"Linguist"
+	message
+		word
+			"Many scientists are currently [working on, engineering] a translation device that will allow direct [communication, dialog] with the Korath in their own language. It is [exciting, satisfying] to be part of this effort.
+			
+news "Wanderers and Aliens"
+	to show
+		has "Wanderers Conversation: offered"
+	location
+		government "Wanderer"
+		not attributes "evacuation"
+	name
+		word
+			"Wanderer merchant captain"
+			"Wanderer captain"
+			"Wanderer civilian"
+			"Wanderer"
+	message
+		word
+			"Long ago, we [rebirthed, cleansed] a group of worlds in the south. I wonder if any [aliens, sentient beings] live there now?"
+			"It is a shame we must [fight, defend against] the Hai. I wish we had the opportunity to [meet, socialize with] their peaceful brethren."
+			"Alien societies [fascinate, interest] me. I [hope, desire] for more opportunities to meet with them."
+	name
+		word
+			"Wanderer linguist"
+			"Linguist"
+	message
+		word
+			"The Hai language is [complicated, convoluted], but learning it has yielded much [insight, knoweldge] about the Hai and [aliens, non-Wanderers] in general."
```

Comment:
Typo + the wanderers are meant to be well old. I doubt that the Hai language provides much insight to anyone but the Hai that they couldn't have thought about from meeting other aliens in the past.
```suggestion
			"The Hai language is [complicated, convoluted], but learning it has yielded much [insight, knowledge] about the Hai in general."
```